alerting/src/test/kotlin/org/opensearch/alerting/MonitorRunnerServiceIT.kt

```diff
@@ -1321,6 +1323,71 @@ class MonitorRunnerServiceIT : AlertingRestTestCase() {
         )
     }
 
+    fun `test bucket-level monitor with findings enabled`() {
```

Comment:
Lets validate other scenarios as well:
1. Where findings will not be created - such as compound aggregation.
2. Findings index unavailable, or exception in index.